src/EPR.RegulatorService.Facade.API/Controllers/ReprocessorExporter/Registrations/RegistrationsController.cs

```diff
@@ -0,0 +1,87 @@
+﻿using System.Net;
+using Asp.Versioning;
+using EPR.RegulatorService.Facade.API.Constants;
+using EPR.RegulatorService.Facade.API.Controllers.ReprocessorExporter.Registrations;
+using EPR.RegulatorService.Facade.Core.Constants;
```

Comment:
Removed unused using statement:

```suggestion
using EPR.RegulatorService.Facade.API.Constants;
using EPR.RegulatorService.Facade.Core.Constants;
```